contents/faq.md

```diff
@@ -19,59 +25,64 @@ The key difference is that you can self-host the platform, and it's open source.
+* Data visualization and product experimentation tools, such as:
+	* [Feature-rich Toolbar](/docs/features/toolbar)
+	* [Feature Flags](/docs/features/feature-flags)
+	* [Session Recording](/docs/features/session-recording)
+	* [Plugins](/docs/features/plugins)
 
-* We have frontend event capture using a snippet that goes inside your HTML ```<head></head>``` tags.
-* For any other events, we have:
-	* [Pre-built libraries](/docs/integrations)
-	* [An API](/docs/api/overview) for anything else
+You can read more about our available features on the dedicated [Product Features page](/product-features).
 
 ### Does it work for mobile and web?
 
 Yes. We support any kind of application.
 
+### Does it work with my stack?
+
+Yes. PostHog can be used via:
+
+- An HTML snippet that goes inside your ```<head></head>``` tags.
+- One of our [15+ integrations](/docs/integrations) covering the main programming languages and tools
+- Our [API](/docs/api/overview)
+
+This means that it doesn't matter what your stack is - you can use PostHog with it.
+
 ### Is this supported properly?
 
-Yes. We are in very [active development](https://github.com/PostHog/posthog/graphs/commit-activity). PostHog is a [well-funded](/handbook/investors) project with [thousands of stars](https://github.com/PostHog/posthog/stargazers) on GitHub and dozens of PRs being worked on weekly.
+Yes. We are in very [active development](https://github.com/PostHog/posthog/graphs/commit-activity). 
+
+PostHog is a [well-funded](/handbook/investors) project with [thousands of stars](https://github.com/PostHog/posthog/stargazers) on GitHub, and dozens of PRs being worked on weekly, by both our [core team](/handbook/company/team) and our active community.
 
-# About the Software
+## About the Software
 
 ### How popular is the software?
 
 Very. We have thousands of users, over [3k stars](https://github.com/PostHog/posthog/stargazers) on GitHub (even though the project only started in January 2020), and [supportive investors](/handbook/investors).
 
 ### Is the software updated regularly?
 
-Yes. We release new versions about every two weeks and have a world-class team working daily on making the product better. It's [easy to update](docs/upgrading-posthog), and the most recent versions of the software will alert you to new updates within the application.
+Yes. We release new versions about every two weeks and have a world-class team working daily on making the product better. It's [easy to update](docs/upgrading-posthog), and the software will alert you to new updates within the application.
 
-Pro-tip: follow us on [Twitter](https://twitter.com/PostHogHQ) / [Slack](/slack) to see our latest features.
+Pro-tip: follow us on [Twitter](https://twitter.com/PostHogHQ) or join our [Slack](/slack) to keep up with our latest features.
 
 ### How many companies use it / How many well-known companies use it?
 
 We have thousands of users including several very large enterprises.
```

Comment:
I'd get rid of "several", as I'd think "oh, like 2" whereas there are significantly more - >10 would be my guess

Reply:
"multiple"?

Reply:
"various"?

Reply:
Ah or just removed it altogether I guess was what you're saying